Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/core/ConceptEntry.java

```diff
@@ -340,12 +350,36 @@ public boolean equals(Object obj) {
         return true;
     }
 
-    public String getModifiedUser() {
-        return modifiedUser;
+ public List<ChangeEvent> getChangeEvents() {
+        return changeEvents;
+    }
+
+    public void setChangeEvents(List<ChangeEvent> changeEvents) {
+        this.changeEvents = changeEvents;
+    }
+
+    public String getLastModifiedUser() {
+        return lastModifiedUser;
+    }
+
+    public void setLastModifiedUser(String lastModifiedUser) {
+        this.lastModifiedUser = lastModifiedUser;
+    }
+
+    public String getLastCreatedUser() {
+        return lastCreatedUser;
+    }
+
+    public void setLastCreatedUser(String lastCreatedUser) {
+        this.lastCreatedUser = lastCreatedUser;
+    }
+
+    public String getDeletedUser() {
+        return deletedUser;
     }
 
-    public void setModifiedUser(String modifiedUser) {
-        this.modifiedUser = modifiedUser;
+    public void setDeletedUser(String deletedUser) {
+        this.deletedUser = deletedUser;
     }
 
 }
```

Comment:
Your ConceptEntry should provide a method to add a change event. The method would also set the last modified user field, creator field, etc.